packages/neo-one-website/src/components/docs/DocFooter.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Box, Grid, styled } from 'reakit';
+import { prop } from 'styled-tools';
+import { AdjacentInfo } from '../../utils';
+import { AdjacentLink } from './AdjacentLink';
+
+interface Props {
+  readonly next?: AdjacentInfo;
+  readonly previous?: AdjacentInfo;
+}
+
+const Footer = styled(Grid)`
+  grid-template: 'prev next';
+  justify-content: space-between;
+  padding: 16px;
+  padding-right: 380px;
```

Comment:
Remove - this should be inherent to the layout. Padding shouldn't be used to layout items. Instead, you should just use a grid -
```
  grid:
    'docs sidebar' auto
    'footer sidebar' auto
    / 1fr 380px
```